docs/Migration-2.4.md

```diff
@@ -34,6 +34,13 @@ Version `2.4.x` introduces a new, extensible **Pre‑approval UI Template** and
 7. **New RejectionReason**
     - New rejection reason `PREAPPROVAL` indicates the user cancelled the operation during the Pre-Approval flow.
 
+8. **MobileTokenData Builder**
+
+	-	Introduced MobileTokenData.Builder, a helper for composing structured data attached to operations during authorization or rejection.
+	-	Supports both generic key–value entries `put(key, value)` and structured records such as WMTPreApprovalScreensRecorder.
+	- 	Records subclass the abstract base class WMTMobileTokenDataRecord, which defines common behavior for building, resetting, and serializing record data.
+	-	Once build() is called on a record, it becomes immutable and is attached to the builder, builder.build() produces the final [String: Encodable] map for operation.mobileTokenData.
```

Comment:
`it becomes immutable and is attached to the builder` - this sentence does not make sense.

The builder does not become immutable - it produces a result object, which can definitely be mutated if the caller chooses to do so - it's just a simple dictionary object